//
// Created by skyjohn on 2020/7/24.
//

#ifndef INC_1_MYPOWWINDOW_H
#define INC_1_MYPOWWINDOW_H
#include "ui_myPowWindow.h"
#include "../../../calc/calculator.h"
using Ui::myPowWindowUi;
class MyPowWindow:public myPowWindowUi {
public:
    explicit MyPowWindow(QWidget *widget);

};


#endif //INC_1_MYPOWWINDOW_H
